relay/http.go

```diff
@@ -26,6 +27,13 @@ type HTTP struct {
 	backends []*httpBackend
 }
 
+const (
+	DefaultHTTPTimeout      = time.Second * 10
```

Comment:
I know it doesn't change the value, but I generally prefer to see `10 * time.Second` so that it reads more like `10 seconds` than `second times 10`